src/lib.rs

```diff
@@ -611,7 +616,7 @@ pub fn print_panic_info(pi: &PanicInfo, s: &mut Settings) -> IOResult {
     }
 
     if s.verbosity >= Verbosity::Medium {
-        print_backtrace(backtrace::Backtrace::new(), s)?;
+        print_backtrace(&backtrace::Backtrace::new(), s)?;
```

Comment:
pretty sure its safe to pass a reference to this unbound temporary object

Reply:
Yes, Rust wouldn't let you do it outside of an unsafe block if it weren't safe.